orttraining/orttraining/python/training/ortmodule/experimental/hierarchical_ortmodule/_hierarchical_ortmodule.py

```diff
@@ -1,15 +1,22 @@
 # Copyright (c) Microsoft Corporation. All rights reserved.
 # Licensed under the MIT License.
 # debug_options.py
+import tempfile
 import torch
-from . import ORTModule
+from ... import ORTModule
 
 
-class HierarchalORTModule(torch.nn.Module):
+class HierarchicalORTModule(torch.nn.Module):
+    # This class recursively wraps sub-modules of "self._wrapped_module"
+    # as ORTModule whenever it can. The actual wrapping happens in its
+    # first "forward(...)" call because we need to call Pytorch-to-ONNX
+    # exporter with actual inputs. Afterwards, supported computation
+    # will be delegated to ORT and unsupported computation is still
+    # done by Pytorch.
```

Comment:
```suggestion
    '''Recursively wraps submodules of `module` as ORTModule whenever possible
    
    Similarly to ORTModule, the actual wrapping happens in its first `forward` call during Pytorch-to-ONNX export.
    Supported computation is delegated to ONNX Runtime and unsupported computation is still done by PyTorch.

    Args:
        module (torch.nn.Module): User's PyTorch module that HierarchicalORTModule specializes
```
Use of docstring is encouraged for public classes, even if they are experimental